configuration/backends/smatch/smatch-wrapper.sh

```diff
@@ -114,8 +114,11 @@ function run_smatch
   logwrapper "${TOOLPREFIX}"SMATCH"${TOOLSUFFIX}" "smatch $SMATCH_EXTRA_ARG $@"
   # in case the source file is analyzed multiple times, collect all messages
   SMATCH_STATUS=0
-  smatch $SMATCH_EXTRA_ARG "$@" "${source_files[@]}" "${header_files[@]}" &>> "$rf" || SMATCH_STATUS=$?
 
+  # do detour via cgcc
+  export CHECK="smatch $SMATCH_EXTRA_ARG "
+  cgcc -no-compile "$@" "${source_files[@]}" "${header_files[@]}" >> "$rf" 2>> >> "$rf".err || SMATCH_STATUS=$?
```

Comment:
Is `2>> >>` (duplicate `>>`) intentional?

Reply:
oh, missed that, will fix. thanks!